core/Cargo.toml

```diff
@@ -137,7 +138,8 @@ sha2 = "0.10.8"
 snap = "1.1.0"
 speedb = { version = "0.0.4", features = ["lz4", "snappy"], optional = true }
 storekey = "0.5.0"
-surrealml-core = { version = "0.0.7", optional = true }
+ml1 = { version = "0.0.7", optional = true, package = "surrealml-core" }
```

Comment:
Could we maybe call these `surrealml-core-v1`? That way the crate name is clear, and the appendix is a separate string

Reply:
Can I do that in a followup PR so as not to invalidate the current approvals?